src/sentry/eventstream/base.py

```diff
@@ -34,26 +35,26 @@ class EventStream(Service):
 
     def _dispatch_post_process_group_task(
         self,
-        event,
-        is_new,
-        is_regression,
-        is_new_group_environment,
-        primary_hash,
-        skip_consume=False,
-    ):
+        event_id: str,
```

Comment:
Minor comment: Would it be better to create a namedtuple or dataclass of the members of event payload we use and pass it around. That might be easier to maintain going forward

Reply:
I'm going to leave it as is. I agree there's a lot of arguments but it's unclear to me that these 3 (project id, group id, event id) are actually a logical set and belong in the event payload any more than the other arguments here (like primary hash) which also typically is a property of an event.